engine/core/src/main/java/org/datacleaner/job/runner/AnalysisJobMetricsImpl.java

```diff
@@ -45,6 +46,7 @@ public ComponentMetrics getComponentMetrics(ComponentJob componentJob) {
         return new ComponentMetricsImpl(rowProcessingMetrics, componentJob);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Unfortunately the AnalysisJobMetrics (and also RowProcessingMetrics btw) has methods on it's interface that is starting to bit-rot a bit. Getting tables __kinda__ makes sense, but is certainly not trivial to read because some tables might represent output data streams. That's why the method was deprecated on the publishers class, but not yet here on the metrics class because the metrics are more direct interface towards the outside world.